.editorconfig

```diff
@@ -0,0 +1,9 @@
+# see https://editorconfig.org/
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_style = tab
```

Comment:
Perhaps I'm misunderstanding. Doesn't this enforce tabs for indents (instead of spaces)? As far as I can tell we always using two spaces, it this is being enforced by Rubocop.

Reply:
Thanks Joe, these are just the [defaults recommended by the sublime text plugin](https://github.com/sindresorhus/editorconfig-sublime#example-file) (that I use). We can change them, which is why i wanted everyone to review but I am not sure what effect it will have on ST.

The ST plugin does not support `tab_width` "intentionally" either (does not say why). 

In ST you typically define your tab width (and can change it easily, but is two spaces generally and should probably stay that way for `.rb` files at least) and then indenting is done using the tab key.

How do you guys indent?

Reply:
@naseberry you could add a commit for `.js` and `.css` files specifically if you wanted. These are currently the fallbacks.

Reply:
on tab issues it is worth looking at the [definition of indent_style](https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#indent_style)

Reply:
@jsugarman suggest we change `indent_style` to space and define the `indent_size` as 2

This would be the preferred formatting for JS, SCSS and most other file type.

My experience in the past with an indent style of tab is that some IDE/Editors have a default size of 4, thereby causing formatting issues between engineers and their preferred IDE/Editor.

Reply:
👍 we can do that and see how it plays.

Reply:
thats been applied now